### ISSUE TITLE
[Android] AutoSuggesBox popup opens showing only the first item

### ISSUE THREAD

**Issue by jeromelaban** [unoplatform/uno#1625]
## Current behavior

When opening an `AutoSuggestBox` popup on android, the list opens up on the first item only. Interacting with the list shows the full list.

## Expected behavior

The list should show the proper number of items initially.

## How to reproduce it (as minimally and precisely as possible)

Use the Android PeoplePicker from the community toolkit: https://github.com/unoplatform/Uno.Graph-Controls

## Environment

Nuget Package:

Package Version(s):

Affected platform(s):
- [ ] iOS
- [x] Android
- [ ] WebAssembly
- [ ] Windows
- [ ] Build tasks

Visual Studio
- [ ] 2017 (version: )
- [ ] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins
- [ ] Resharper (version: )

**Comment by davidjohnoliver** [comment]
@LHuss @carldebilly do you know if this issue is still valid?

**Comment by jeromelaban** [comment]
I have not looked into that recently.

**Comment by duke7553** [comment]
In my testing of this scenario on a Pixel 3a with the latest Uno.UI--3.7.4--every suggestion seems to be displayed properly even after the AutoSuggestBox lost focus and regained it. I also tested setting ItemsSource initially in XAML and all suggestion were visible when the app loaded.

@jeromelaban This should be fixed now, afaik. I couldn't get the PeoplePicker repro app to build, so I built my own to test the issue.
